Jenkinsfile

```diff
@@ -40,6 +40,6 @@
 // I.e. for testing library changes
 //@Library(value="pipeline-lib@your_branch") _
 
-// centos8: No matching package to install: 'fio-devel'
-packageBuildingPipelineDAOS(['distros': ['centos7', 'centos8', 'leap15',
+// el8: No matching package to install: 'fio-devel'
+packageBuildingPipelineDAOS(['distros': ['el7', 'el8', 'leap15',
```

Comment:
It's only `centos8` that's being migrated to `el8`.  `centos7` is staying as it is.

```suggestion
packageBuildingPipelineDAOS(['distros': ['centosl7', 'el8', 'leap15',
```

Reply:
I just removed for this PR

Reply:
put back but still seeing failure in rpm test stage